README.md

```diff
@@ -512,3 +531,21 @@ SignalFx::Tracing::Instrumenter.configure do |p|
 end
 ```
 
+## Configuring the Logger
+
+The logger, based on the [Ruby Logger](https://ruby-doc.org/stdlib-2.4.0/libdoc/logger/rdoc/Logger.html), can be configured by setting the following environment variables:
+
+| Environmental Variable Name   | Description           |  Default             |
+|:------------------------------|:----------------------|:-------------------- |
+| `SIGNALFX_LOG_DIR`            | The path to the desired directory  where the instrumentation logs will be written. Besides customized paths, an output option also includes `STDOUT` for logging directly to the console.| `/var/log/signalfx` |
+| `SIGNALFX_LOG_FILE`           | The desired file name for writing the logs | `signalfx-ruby-tracing.log`|
+| `SIGNALFX_LOG_SHIFT_AGE`      | The desired number of old log files to keep, or frequency of rotation. Options include: `daily`, `weekly` or `monthly`)  | `0`    |
+| `SIGNALFX_LOG_SHIFT_SIZE`     | The desired maximum size of log files (this only applies when. A new one would be created when the maximum is reached. | `1048576` (1 MB)  |
+| `SIGNALFX_LOG_LEVEL`          | The severity criteria for recording logs (from least to most severe). Options: `debug`, `info`, `warn`, `error`, `fatal`, `unknown`  | `WARN` |
```

Comment:
nit: the default would be `"warn"` from this options list since they aren't providing a symbol.